ide-support/revsaveasiosstandalone.livecodescript

```diff
@@ -618,14 +618,19 @@ private command revSaveAsMobileStandaloneMain pStack, pAppBundle, pTarget, pSett
       put revGetMobileProfileInfo(pSettings["ios,profile"]) into tProfileInfo
       
       -- Get the list of valid identities that are available
-      local tValidIdentities
+      local tValidIdentities,tValidIdentitiesHEX
       get shell("/usr/bin/security -q find-identity -v")
       if the last line of it contains "valid identities found" then
          delete the last line of it
+         -- "it" now is of the form:
+         -- 1) Certificate1IdHEX "Certificate1NameString"
+         -- 2) Certificate2IdHEX "Certificate2NameString"
          repeat for each line tIdentity in it
             put char 2 to -2 of the last word of tIdentity & return after tValidIdentities
+            put char 1 to -1 of the second word of tIdentity & return after tValidIdentitiesHEX
```

Comment:
It would also probably simplify the implementation here to use an array:
```
put char 1 to -1 of the second word of tIdentity into tValidIdentities[char 2 to -2 of the last word of tIdentity]
...
-- remove the delete last char lines
...
if tValidIdentities[tIdentity] is not empty then
   put tValidIdentities[tIdentity] into tCertificate
end if
```